testing/testhost.cc

```diff
@@ -0,0 +1,6 @@
+#include <gtest/gtest.h>
+
```

Comment:
If you link iwth gtest::ain, this should be useless

Reply:
Removed testhost.cc